.travis.yml

```diff
@@ -52,6 +53,10 @@ matrix:
     - rvm: 2.1.0
       env: PUPPET_GEM_VERSION="~> 3.8.0" CHECK=build DEPLOY_TO_FORGE=yes
     - rvm: 1.9.3
+      env: PUPPET_GEM_VERSION="~> 4.9.0"
+    - rvm: 1.9.3
+      env: PUPPET_GEM_VERSION="~> 4"
+    - rvm: 1.9.3
```

Comment:
Seems these two were not excluded. I believe you need to add "CHECK=test" to both of them